### DISCUSSION TITLE
Missing receiver AUX channel map

### DISCUSSION THREAD

**Discussion by serhii-bilyk-intellias** [iNavFlight/inav-configurator#1906] [Q&A] [answered]
This is my receiver map:
![image](https://github.com/iNavFlight/inav-configurator/assets/144336942/73e6654f-5d19-4bd5-b520-534a6a749575)
After I flashed speedybeef405 flight controller and changed PORST I dont cee channel map for my receiver and cant bind my drone
Inav 6.1.1 firmvare

**Comment by MrD-RC** [comment, marked as answer]
You haven’t set up a receiver type. See **Receiver Mode** _None_.

Binding has nothing to do with INAV. That is between your transmitter and receiver.
